pkgmgmtutils/utils.go

```diff
@@ -0,0 +1,150 @@
+package pkgmgmtutils
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+var (
+	ErrNoSha256sFound = errors.New("did not find all needed sha256 files")
```

Comment:
Typical pattern in go as I've seen it would be:
```
errNoSha256sFound = errors.New("did not find all needed sha256 files")
var IsNoSha256sFoundErr(err error) bool {
    return err == errNoSha256sFound
}
```
Totally optional though

Reply:
Tweaked Error code to match these conventions https://github.com/golang/go/wiki/Errors. Happy to update to better match Solo-io standards; just need to know what those are. :-)